src/SonarScanner.MSBuild.Common/ProcessRunnerArguments.cs

```diff
@@ -216,17 +216,22 @@ private static string ShellEscape(string argLine)
             var sb = new StringBuilder();
             foreach (var c in argLine)
```

Comment:
Could you please cover the change in logic with Unit tests?

Reply:
@csaba-sagi-sonarsource done.